iceoryx_binding_c/include/iceoryx_binding_c/types.h

```diff
@@ -33,13 +33,13 @@
     (144 + numberOfAttachments * 168 - 8 * (((numberOfAttachments + 1) / 2) - 1))
 #elif defined(_WIN32)
 #define CALCULATE_STORAGE_SIZE_FOR_LISTENER(numberOfAttachments)                                                       \
-    (168 + numberOfAttachments * 176 - 8 * (((numberOfAttachments + 1) / 2) - 1))
+    (168 + numberOfAttachments * 192 - 8 * (((numberOfAttachments + 1) / 2) - 1))
```

Comment:
May work but is not instilling confidence ... but it is not configurable for the user right?

Reply:
No, it is not configurable. We could also just make it a fixed number. With this it's easier to adapt the number of attachments and there are tests to ensure that the storage fits the actual object. The proper solution would be a proper memory management but that is out of scope for v2.0